tests/functional/fixtures/ext/foo/bar/config/services.yml

```diff
@@ -14,7 +14,13 @@ services:
         class: foo\bar\event\permission
         tags:
             - { name: event.listener }
+
     foo_bar.listener.user_setup:
         class: foo\bar\event\user_setup
         tags:
             - { name: event.listener }
+
+    foo_bar.listener.template_event_order:
+        class: foo\bar\event\template_event_order
+        tags:
+            - { name: event.listener }
```

Comment:
Shouldn't this also include `foo_bar.listener.template_event_order_lower`?

Reply:
Nope, "lower" one just being copied to a listener file temporarily. Foo is not touched at all just bar priority changed.

Reply:
Ah, I see the `copy` command in the `assertTrue` now. I'll resolve this one for now but consider my comments about moving one of the tests to `foo_foo`